### DISCUSSION TITLE
french version

### DISCUSSION THREAD

**Discussion by ghost** [leihs/leihs#1670] [Q&A / Community Support]
Hi,

We are considering implementing "leihs" for the management of audiovisual equipment at our school of applied arts.

Is a .po file or an equivalent file available in Git to provide a translation?

Thank you for your reply.

Best regards
Filippo

**Comment by neoheat** [comment]
Hi @filnug 
Please see the documentation on this topic: https://github.com/leihs/leihs/wiki/Translation-Guide
best, Lukas

**Reply by neoheat** [reply]
addition: the borrow app uses a clojure file found in 'src/common/leihs/borrow'
